sales_team_security/models/res_partner.py

```diff
@@ -40,3 +41,54 @@ def _onchange_parent_id_sales_team_security(self):
         """
         if self.parent_id and self.parent_id.team_id and not self.team_id:
             self.team_id = self.parent_id.team_id.id
+
+    @api.depends('company_type', 'child_ids', 'child_ids.user_id')
+    def sync_followers_in_childs(self):
+        for item in self.filtered(lambda x: x.company_type == 'company'):
+            partners = self.env['res.partner'].search([
```

Comment:
I don't see here any extraction of the delivery / invoicing contacts. There's a method for that in the partner.

Reply:
Function apply to `child` ids